$OpenBSD: patch-tests_runtests,v 1.2 2008/08/28 19:55:20 kili Exp $

Don't use bash.

If there are unexpected results, bail out.

--- tests/runtests.orig	Thu Nov 22 11:32:51 2007
+++ tests/runtests	Fri Nov 23 21:14:57 2007
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # Run haskell test suites.
 
 if uname -a | grep -i CYGWIN >/dev/null
@@ -71,6 +71,7 @@ run () {
   echo >&2 "                  / of which $hmakeok known / $hmakebad bad"
   echo >&2 "$outok outputs match / $outbad outputs differ"
   echo >&2 "$errok errors  match / $errbad errors  differ"
+  return $((hmakebad || outbad || errbad))
 }
 
 clean () {
